packages/fast-components-styles-msft/src/listbox-item/index.ts

```diff
@@ -0,0 +1,113 @@
+import designSystemDefaults, {
```

Comment:
@chrisdholt @nicholasrice do you think this should be a msft style component? @scomea what is the use case for this not just being a base component?

Reply:
I think we want to maximize the flexibility of what a listbox item is.  Our version uses displaystring, some other might not and choose to base visuals on children and not use displaystring at all, for example.

Reply:
I think I'm hesitant here because I don't know that we have a MSFT listbox (non-select) which is styled. Is this primarily to support select? If so, it seems like an odd abstraction to me.

Reply:
How is this different from context menu item?  This is essentially the same pattern.

Reply:
We have a design and style for context menu item. I don't know that we have a design for *just* as listbox. @bheston can advise on this from a design standpoint. We could have a "select item" here I think - as we have a design for select which happens to be composed of listbox items. I just don't know if we have a default style for a functional listbox. While listboxes can be styled but I don't think we have a default style or a listbox design system component. We have select which is styled and functionally made up of listbox, but those are different things I think.

Reply:
I had considered having an "option" componentb (and context) instead of "listbox item" to try to be more generic.

Reply:
I'm still not seeing the use case or the request from design for this to be a component, let's put it this way, just because we have a tabs component does not mean that there is an msft styled tabs component (we do, pivot, but that's not the point). Things are not necessarily 1:1 for base and msft.

Consider that the reason tabs was added as a base component was because it is an established functional pattern. If there was no need for a tabs component styled for msft, we simply would never add it. However tabs can still be used as a pattern for an msft component like carousel. Or there could simply be the case where we never need to use tabs in msft because there is no corresponding component, that's OK too.

I believe, and correct me if I'm wrong, the only requirement we have right now is for a msft select.

Reply:
So perhaps we don't have listbox as a styled component and the styled "select" component uses listbox base, and then instead of a styled listbox item we have a styled component called "select option" which wraps the base listbox item?

Reply:
See the "Select Base" PR for what the above proposal looks like.